yt_dlp/extractor/youtube/_video.py

```diff
@@ -4262,8 +4296,15 @@ def process_language(container, base_url, lang_code, sub_name, client_name, quer
                 elif info.get('view_count') is None:
                     info['view_count'] = vc
 
+            if self._preferred_lang:
+                info['title'] = self._get_text(vpir, 'title') or info['title']
+
         vsir = get_first(contents, 'videoSecondaryInfoRenderer')
         if vsir:
+            if self._preferred_lang:
+                info['description'] = traverse_obj(
```

Comment:
Ditto on moving this to be the value for `translated_description` like the title, rather than overwriting the current `description`.